package.json

```diff
@@ -64,6 +65,7 @@
     "ajv": "^8.12.0",
     "ajv-draft-04": "^1.0.0",
     "ajv-formats": "^2.1.1",
```

Comment:
check on `aws-sdk` & `pprof` new dependencies too

Reply:
pprof dependency has been long ago removed & instead we are using "@datadog/pprof": "^3.1.0" now - share the PR that suggested this change

Reply:
https://github.com/rudderlabs/rudder-transformer/pull/2498/files

Reply:
These are old PR's after which few dependencies got cleaned up - https://github.com/rudderlabs/rudder-transformer/commit/397277b4328f94ed6b732a2f61677eaba5e7bda0
Remove the new dependencies added in this PR